mpas_analysis/ocean/climatology_map_bgc.py

```diff
@@ -0,0 +1,122 @@
+        -------
+        Phillip J. Wolfram
+        """
+        # call the constructor from the base class (AnalysisTask)
+        super(ClimatologyMapBGC, self).__init__(
+                config=config, taskName='climatologyMapBGC_' + afieldName,
+                componentName='ocean',
+                tags=['climatology', 'horizontalMap', 'BGC', afieldName])
+
+        iselValues = {'nVertLevels': 0}
+
+        sectionName = 'climatologyMapBGC'
+
+        climStartYear = config.getint('climatology',
+                                      'startYear')
+        climEndYear = config.getint('climatology',
+                                    'endYear')
+
+        # read in what seasons we want to plot
+        seasons = config.getExpression(sectionName, 'seasons')
+
+        if len(seasons) == 0:
+            raise ValueError('config section {} does not contain valid list '
+                             'of seasons'.format(sectionName))
+
+        comparisonGridNames = config.getExpression(sectionName,
+                                                   'comparisonGrids')
+
+        if len(comparisonGridNames) == 0:
+            raise ValueError('config section {} does not contain valid list '
+                             'of comparison grids'.format(sectionName))
+
+        # the variable mpasFieldName will be added to mpasClimatologyTask
+        # along with the seasons.
+        remapClimatologySubtask = RemapMpasClimatologySubtask(
+            mpasClimatologyTask=mpasClimatologyTask,
+            parentTask=self,
+            climatologyName=afieldName,
+            variableList=[ampasFieldName],
+            comparisonGridNames=comparisonGridNames,
+            seasons=seasons,
+            iselValues=iselValues)
+
+        for comparisonGridName in comparisonGridNames:
+            for season in seasons:
+                # make a new subtask for this season and comparison grid
+                subtask = PlotClimatologyMapSubtask(self, season,
+                                                    comparisonGridName,
+                                                    remapClimatologySubtask,
+                                                    None,
+                                                    refConfig)
+
+                subtask.set_plot_info(
+                        outFileLabel=afieldName,
+                        fieldNameInTitle=afieldName,
+                        mpasFieldName=ampasFieldName,
+                        refFieldName=None,
+                        refTitleLabel=None,
+                        unitsLabel=r'mmol m$^{-3}$',
+                        imageCaption='Mean ' + afieldName,
+                        galleryGroup='Sea Surface Biogeochemistry',
+                        groupSubtitle=None,
+                        groupLink=afieldName,
+                        galleryName=afieldName,
+                        diffTitleLabel=None)
+
+                self.add_subtask(subtask)
+        # }}}
+    # }}}
```

Comment:
@pwolfram, you will want to add a `setup_and_check()` function here to check if the right config options are enabled for BGC to work and/or to check if the necessary fields are available in the simulation results.